lib/phoenix/endpoint/supervisor.ex

```diff
@@ -327,19 +329,26 @@ defmodule Phoenix.Endpoint.Supervisor do
   """
   @invalid_local_url_chars ["\\"]
 
-  def static_path(_endpoint, "//" <> _ = path) do
+  def static_path(endpoint, path) do
```

Comment:
Is this function ever called now? What called `static_path/2` before is now calling `static_lookup/2`.

Reply:
I believe it is called in some tests, and since it is public I didn't want to remove it without at least giving a deprecation notice beforehand.

Reply:
It is not public. The whole module is doc false. We should update the docs to call `static_lookup` instead. :)

Reply:
Will update